From: Caleb Connolly <caleb.connolly@linaro.org>
Date: Thu, 27 Jul 2023 21:33:39 +0100
Subject: sdm845.config: make typec builtin

The QMP combo phy (USB superspeed + DP alt mode) now depends on
CONFIG_TYPEC, enable it as builtin so that we can get USB without
modules.

Signed-off-by: Caleb Connolly <caleb.connolly@linaro.org>
---
 arch/arm64/configs/sdm845.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/sdm845.config b/arch/arm64/configs/sdm845.config
index 81c1eef..4ff2121 100644
--- a/arch/arm64/configs/sdm845.config
+++ b/arch/arm64/configs/sdm845.config
@@ -155,6 +155,7 @@ CONFIG_QCOM_SPMI_ADC5=y
 CONFIG_PHY_QCOM_QMP=y
 CONFIG_PHY_QCOM_QUSB2=y
 CONFIG_PHY_QCOM_QMP_UFS=y
+CONFIG_TYPEC=y
 CONFIG_PHY_QCOM_QMP_COMBO=y
 CONFIG_LEDS_CLASS_FLASH=y
 CONFIG_TCP_CONG_ADVANCED=y
